packages/components/src/interfaces/internal/GoogleTagManager.ts

```diff
@@ -0,0 +1,21 @@
+import type { ScriptComponentType } from "~/types"
+
+export interface GoogleTagManagerHeaderScriptProps {
+  gtmId: string
+  ScriptComponent?: ScriptComponentType
+}
+
+export interface GoogleTagManagerHeaderProps {
+  siteGtmId: string | undefined
+  isomerGtmId: string | undefined
```

Comment:
nit: undefined can use the `?` operator

```suggestion
  siteGtmId?: string
  isomerGtmId?: string
```